src/error.rs

```diff
@@ -1,10 +1,17 @@
 use crate::Result;
 use snafu::{
 	Backtrace,
+	GenerateBacktrace,
 	ResultExt,
 	Snafu,
 };
 
+pub fn unwrap_field<T>(x: Option<T>) -> Result<T> {
```

Comment:
You can replace this by importing `snafu::OptionExt` and calling `.context(error::MissingData)?;` instead.